content/applications/finance/accounting/payables/pay/multiple.rst

```diff
@@ -2,146 +2,46 @@
-
-Once you have validated the payment, the system will automatically
-reconcile the payment with the bill, and set the bill as **Paid**. The
-system will also generate a move from the payment account and reconcile
-it with the expense transaction.
-
-Pay several bills altogether
-----------------------------
-
-In order to illustrate the process thoroughly, create at least 2 more
-bills following the above standing guide. **Make sure all bills come
-from the same vendor.**
-
-.. image:: multiple/multiple05.png
-  :align: center
-
-In the Vendors Bills, select the new bills you have just created by
-checking the box next to each of them. In the Action menu located in the
-middle of the page, click on **Register Payment**.
-
-.. image:: multiple/multiple06.png
-  :align: center
-
-Insert the details of the payment. The system calculated the total
-amount for both bills, but you can modify it freely. Click on **Validate**.
-
-Record the payment, reconcile afterwards
-----------------------------------------
-
-You can also reconcile a payment with bills after the payment has been
-recorded.
-
-First, we need to create a payment
-
-This will handle from :menuselection:`Dashboard --> Bank journal -->
-More Option --> Send Money`
-
-.. image:: multiple/multiple07.png
-  :align: center
-
-Creating payment order with check payment method. Selecting related
-Vendor and amount which remain to pay. After filling all details, we
-will confirm the payment order which will generate payment transaction
-with the system.
-
-.. image:: multiple/multiple08.png
-  :align: center
-
-As you can see, bill payment status show what is posted and what is
-remaining to reconcile.
-
-After receiving bank statement from the bank with payment detail, you
-can reconcile the transaction from the Dashboard. It will automatically
-map the transaction amount.
+Odoo offers the possibility of grouping multiple bills' payments into one, facilitating the
+reconciliation process.
 
 .. seealso::
+   - :doc:`../supplier_bills/manage`
    - :doc:`../../bank/reconciliation/use_cases`
 
-Partial payments of several supplier bills
-==========================================
-
-How to pay several supplier bills having cash discounts at once?
-----------------------------------------------------------------
-
-You already learned how to pay bills in various way but what about
-partial payment? We are taking another example where we will do partial
-payment for various bills.
-
-We are creating multiple bills and partially pay them through bank
-statements.
-
-We are adding payment terms which allow some cash discount where vendor
-offer us early payment discount.
-
-.. image:: multiple/multiple09.png
-  :align: center
-
-We are creating the following bills with the assignment of the above
-payment term.
+Group payments
+==============
 
-.. image:: multiple/multiple10.png
-  :align: center
+To register the payment of multiple bills at once, go to :menuselection:`Vendors --> Bills`. Then,
```

Comment:
I think accounting app makes it clear. Because there's the possibility to be in the Settings "app" so-to-say, that way it's clear it's from within the accounting app